README.md

```diff
@@ -58,12 +63,17 @@ or your package manager on Linux.
     **Note:** If you use Chocolatey to install Gstreamer, it does this
     automatically.
 
-3. Download and extract a [prebuilt binary from continuous integration][ci-download]
-(click on the topmost commit for the most recent build).
-
 3. Download and unpack Neolink from the links above.
    1. Note: you can also click on [this link][ci-download] to see all historical builds.
-  You will need to be logged in to GitHub to download directly from the builds page.
+  You will need to be logged in to GitHub to download directly from the builds page (wget doesn't work)
```

Comment:
I think `wget`/`curl` will only not work for the historical build but the links on this readme page will work over `wget`/`curl`